scripts/get_maintainer.py

```diff
@@ -479,7 +485,7 @@ def ferr(msg):
 
     ok_keys = {"status", "maintainers", "collaborators", "inform", "files",
                "files-exclude", "files-regex", "files-regex-exclude",
-               "labels", "description"}
+               "labels", "description", "tests", "tags"}
```

Comment:
tags in this file can be removed right?

Reply:
can, but leaving it does not hurt, I might have a usecase for this to improve coverage as we move forward.

Reply:
ack